config/initializers/gds-sso.rb

```diff
@@ -0,0 +1,13 @@
+GDS::SSO.config do |config|
+  config.user_model   = 'User'
+
+  # set up ID and Secret in a way which doesn't require it to be checked in to source control...
+  config.oauth_id     = ENV['OAUTH_ID']
+  config.oauth_secret = ENV['OAUTH_SECRET']
+
+  # optional config for location of Signon
+  config.oauth_root_url = "http://localhost:3001"
```

Comment:
I think we want to use plek to find signon location here. As per: https://github.com/alphagov/support-api/blob/1281113d26c5edff8af43e1a6b6e410bd574effd/config/initializers/gds-sso.rb#L5

Reply:
fixed